code/datums/components/only_pull_living.dm

```diff
@@ -0,0 +1,14 @@
+/// Component for only letting a living pull other livings
+/datum/component/only_pull_living
```

Comment:
This should be an element it does basically nothing

Reply:
I forgot elements existed